### ISSUE TITLE
-e parameter causing command line parse error on windows

### ISSUE THREAD

**Issue by AndyTeas** [DOSarrest-Internet-Security/alfred#2]
When testing on a Win7 machine, have found that specifying the "-e" command line option, we seem to be getting a fatal error.
here's the output from the test we ran:

---

C:\Java\bin>java -version 
java version "1.7.0_55"
Java(TM) SE Runtime Environment (build 1.7.0_55-b13)
Java HotSpot(TM) 64-Bit Server VM (build 24.55-b03, mixed mode)

REM **************************************************************************************
REM ::When the "-e" param is missing from command line, Alfred behaves as expected and will remove anything older than 24 days (the default): 
REM **************************************************************************************

C:\Java\bin>java -jar ../alfred-0.0.1.jar -i".marvel-*" -d -T"day" 
GENERAL: .marvel-2014.05.12 is 23 days above the cuttoff.
GENERAL: .marvel-2014.05.11 is 22 days above the cuttoff.
GENERAL: .marvel-2014.05.09 is 20 days above the cuttoff.
GENERAL: .marvel-2014.05.08 is 19 days above the cuttoff.
GENERAL: .marvel-2014.05.06 is 17 days above the cuttoff.
GENERAL: .marvel-2014.05.05 is 16 days above the cuttoff.
GENERAL: .marvel-2014.05.02 is 13 days above the cuttoff.
GENERAL: .marvel-2014.05.01 is 12 days above the cuttoff.
GENERAL: .marvel-2014.04.30 is 11 days above the cuttoff.
GENERAL: .marvel-2014.04.29 is 10 days above the cuttoff.
GENERAL: .marvel-2014.04.28 is 9 days above the cuttoff.
GENERAL: .marvel-2014.04.26 is 7 days above the cuttoff.
GENERAL: .marvel-2014.04.25 is 6 days above the cuttoff.
GENERAL: .marvel-2014.04.24 is 5 days above the cuttoff.
GENERAL: .marvel-2014.04.23 is 4 days above the cuttoff.
GENERAL: .marvel-2014.04.22 is 3 days above the cuttoff.
GENERAL: .marvel-2014.04.21 is 2 days above the cuttoff.
GENERAL: Index .marvel-2014.04.17 would have been deleted for being 1 days older than expiry time.
GENERAL: Index .marvel-2014.04.16 would have been deleted for being 2 days older than expiry time.
GENERAL: Index .marvel-2014.04.15 would have been deleted for being 3 days older than expiry time.
GENERAL: Index .marvel-2014.04.14 would have been deleted for being 4 days older than expiry time.
GENERAL: Index .marvel-2014.04.13 would have been deleted for being 5 days older than expiry time.
GENERAL: Index .marvel-2014.04.12 would have been deleted for being 6 days older than expiry time.
GENERAL: Index .marvel-2014.04.11 would have been deleted for being 7 days older than expiry time.
GENERAL: Index .marvel-2014.04.10 would have been deleted for being 8 days older than expiry time.
GENERAL: Index .marvel-2014.04.09 would have been deleted for being 9 days older than expiry time.
GENERAL: Index .marvel-2014.04.08 would have been deleted for being 10 days older than expiry time.
GENERAL: Index .marvel-2014.04.07 would have been deleted for being 11 days older than expiry time.
GENERAL: Index .marvel-2014.04.04 would have been deleted for being 14 days older than expiry time.
GENERAL: Index .marvel-2014.04.03 would have been deleted for being 15 days older than expiry time.
GENERAL: Index .marvel-2014.04.02 would have been deleted for being 16 days older than expiry time.
GENERAL: Index .marvel-2014.04.01 would have been deleted for being 17 days older than expiry time.
GENERAL: Index .marvel-2014.03.31 would have been deleted for being 18 days older than expiry time.
GENERAL: Index .marvel-2014.03.29 would have been deleted for being 20 days older than expiry time.
GENERAL: Index .marvel-2014.03.28 would have been deleted for being 21 days older than expiry time.
GENERAL: Index .marvel-2014.03.27 would have been deleted for being 22 days older than expiry time.
GENERAL: Index .marvel-2014.03.26 would have been deleted for being 23 days older than expiry time.

REM **************************************************************************************
REM ::When the "-e" param is included, Alfred fails to parse the command line (several variations were attempted) 
REM **************************************************************************************

C:\Java\bin>java -jar ../alfred-0.0.1.jar -D"info" -i".marvel-*" -d -T"day" -e7 
FATAL: Error parsing command line options!
usage: alfred
(help text removed for brevity)
Alfred Version: 0.0.1

C:\Java\bin>java -jar ../alfred-0.0.1.jar -D"info" -i".marvel-*" -d -T"day" -e"7" 
FATAL: Error parsing command line options!
usage: alfred
 (help text removed for brevity)
Alfred Version: 0.0.1

C:\Java\bin>java -jar ../alfred-0.0.1.jar -D"info" -i".marvel-*" -T"day" -e7 -d 
FATAL: Error parsing command line options!
usage: alfred
 (help text removed for brevity)
Alfred Version: 0.0.1

C:\Java\bin>java -jar ../alfred-0.0.1.jar -D"info" -i".marvel-*" -T"day" -e"7" -d 
FATAL: Error parsing command line options!
usage: alfred
 -b,--debloom                  Disable Bloom on Indexes
 -B,--bloom                    Enable Bloom on Indexes
 -c,--close                    Close Indexes
 -D,--debug <arg>              Display debug (debug|info|warn|error|fatal)
 -d,--delete                   Delete Indexes
 -E,--expiresize <arg>         Byte size limit  (Default 10 GB)
 -e,--expiretime <arg>         Number of time units old (Default 24)
    --examples                 Show some examples of how to use Alfred
 -f,--flush                    Flush Indexes
 -h,--help                     Help Page (Viewing Now)
    --host <arg>               ElasticSearch Host
 -i,--index <arg>              Index pattern to match (Default _all)
    --max_num_segments <arg>   Optimize max_num_segments (Default 2)
 -o,--optimize                 Optimize Indexes
 -O,--open                     Open Indexes
    --port <arg>               ElasticSearch Port
 -r,--run                      Required to execute changes on
                               ElasticSearch
 -s,--style <arg>              Clean up style (time|size) (Default time)
 -S,--settings <arg>           PUT settings
    --ssl                      ElasticSearch SSL
 -T,--time-unit <arg>          Specify time units (hour|day|none) (Default
                               hour)
 -t,--timeout <arg>            ElasticSearch Timeout (Default 30)
Alfred Version: 0.0.1

**Comment by DOSarrest** [comment]
Hello Andy, sorry about not responding to this issue for a long time. We fixed this issue in the code earlier, but then we ran into some issues with ElasticSearch which took us a long time to resolve.

This issue occurred because we set up grouped options with Apache Commons CLI, and that broke some of the intended functionality.

We have put up 0.0.3 which has the fix for this issue as well as the added functionality of specifying exclude patterns.
